src/main/resources/assets/paths/studyById.yaml

```
@@ -0,0 +1,23 @@
+get:
+  summary: Get Study by Id
+  description: Finds Study and Properties of the Study specified by id.
+  parameters:
+    - name: id
```

The variable here should match what's in api-docs
```suggestion
    - name: studyId
```